jenkins/opensearch/perf-test.jenkinsfile

```diff
@@ -6,7 +6,7 @@ pipeline {
         timeout(time: 10, unit: 'HOURS')
     }
     environment {
-        AGENT_LABEL = 'Jenkins-Agent-al2-x64-c54xlarge-Docker-Host'
+        AGENT_LABEL = 'Jenkins-Agent-al2-x64-m5xlarge-Perf-Test'
```

Comment:
Can you confirm that this agent configuration exists in the current  jenkins set up before we merge?
Thanks!

Reply:
I did check the server for the agent and it does exist. I tried to run a sample test which failed though.
Error - 
```
+ docker pull opensearchstaging/ci-runner:ci-runner-centos7-v1
/var/jenkins/workspace/perf-tests-dev/perf-test-v3@tmp/durable-effa1e97/script.sh: line 1: docker: command not found
```
cc: @peterzhuamazon

Reply:
Yeah I checked and looks like docker installation commands are missing. Is someone is taking care of it? If not I can add it.

Reply:
@kotwanikunal like I mentioned before you dont use docker on that particular host.
You directly use that agent therefore no docker needed.
Thanks.

Reply:
Synced up offline. Resorted to using docker for this use case.